src/Workspaces/Core/Portable/Shared/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -55,6 +55,24 @@ public static bool IsSystemVoid([NotNullWhen(returnValue: true)] this ITypeSymbo
         public static bool IsNullable([NotNullWhen(returnValue: true)] this ITypeSymbol? symbol)
             => symbol?.OriginalDefinition.SpecialType == SpecialType.System_Nullable_T;
 
+        public static bool IsNullable(
+            [NotNullWhen(returnValue: true)] this ITypeSymbol? symbol,
+            [NotNullWhen(returnValue: true)] out ITypeSymbol? underlyingType)
```

Comment:
Normally I'm a big proponent of naming arguments whenever the argument isn't naming itself, but here is one place I really dislike it. It's going to be such a common annotation to get used to and the meaning really is unmistakable since it can't be referring to the parameter.